### ISSUE TITLE
Component - Chocolate box (3 column) 

### ISSUE THREAD

**Issue by twblack88** [makerdao/community-portal#18]
This component  provides a visual grouping of related content e.g. examples of use-cases or benefits

[Figma frame](https://www.figma.com/file/mVblaaFzkqracWycXl9czh/Maker-comm-dev?node-id=486%3A1), see 'off chain' and 'on chain' Governance sections for the structure. 

[More details](https://www.notion.so/Components-e4455d1797f74c48aae3019692a39413#a1eeb7e0d62b4dd2ac52d6121779f3c0)

**Comment by MaximumCrash** [comment]
@isaacpearl (I'm late with the notes on this I apoligize)

Chocolate boxes are simple components, especially since we don't have to be concerned with managing their individual heights. 

This component is a case where we must go against the rules of it's design to ensure it can be scripted flexibly in MDX. What I mean is, while the designs provided Chocolate Box's in examples of 3x2 It should support an infinite amount of containers.

**NOTE: I would recommend using Grid styles for this since they have specific sizes.**

1 Container
- Max Width 245px
- Align Left

2 Containers
- Max Width 245px each
- Align Center / Space Around

3 Containers
- Max Width 245px each
- Align Center / Space Around (by this point it should fill up the width of it's parent container)

![Screen Shot 2020-06-19 at 9 02 58 AM](https://user-images.githubusercontent.com/9157341/85135173-b0464100-b20b-11ea-8774-5d1561afd4e7.png)

4 Containers
- Max Width 245px each
- Align Center

5 - Infinity
Rinse and Repeat 3 Container styles
![Screen Shot 2020-06-19 at 9 04 43 AM](https://user-images.githubusercontent.com/9157341/85135340-eedbfb80-b20b-11ea-90a8-450fe7fd3936.png)


**Acceptance criteria** will be (very similar to Column Comparise):

1. I should be able to script this component in MDX with as little friction to remember as possible. This means that each child that MDX sees should be the equivalent of 1 Chocolate Box. **(See my List/Process components for examples of this)**

**NOTE: Mind the gaps. MDX is picky about line breaks (similarly to MD) on how it recognizes individual children. Notice the line break at the start and end of the component and in-between children.**

```
<Chocolate>

   Choco Box 1

   - Choco Box 2
 
   - Choco Box 3

   1. Choco Box 4
   
    ### Choco Box 5

    <Box>
      Choco Box 6
    </Box>

</Chocolate>
```

A more realistic example would be:

```

<Chocolate>

<Box>
   
<Icon name="stocks" size={3}/>

Measuure community sentiment about issues affecting the MakerDAO ecosystem.

[See example ➡](https://example.com)

</Box>

<Box>
   
<Icon name="scale" size={3}/>

Determine the consensus that something needs to be done in response to a perceived issue.

[See example ➡](https://example.com)

</Box>

</Chocolate>

```

2. No margin top for this component when it comes to spacing around other components. But the individual components do have a margin bottom. 

4. Height differences. From the spec, height differences can be ignored as the tallest will automatically push down the next row.

**Comment by MaximumCrash** [comment]
Closed by #65
